migration-roadmap/badge.md

```diff
@@ -0,0 +1,337 @@
+- `--mod-badge-icon-only-spacing-horizontal`
+- `--mod-badge-icon-spacing-horizontal`
+- `--mod-badge-icon-spacing-vertical-top`
+- `--mod-badge-icon-text-spacing`
+- `--mod-badge-label-icon-color`
+- `--mod-badge-label-spacing-horizontal`
+- `--mod-badge-label-spacing-vertical-bottom`
+- `--mod-badge-label-spacing-vertical-top`
+- `--mod-badge-line-height`
+- `--mod-badge-line-height-cjk`
+- `--mod-badge-outline-background-color-default`
+- `--mod-badge-outline-border-color-accent`
+- `--mod-badge-outline-border-color-informative`
+- `--mod-badge-outline-border-color-negative`
+- `--mod-badge-outline-border-color-neutral`
+- `--mod-badge-outline-border-color-notice`
+- `--mod-badge-outline-border-color-positive`
+- `--mod-badge-outline-label-icon-color`
+- `--mod-badge-subtle-background-color-accent`
+- `--mod-badge-subtle-background-color-blue`
+- `--mod-badge-subtle-background-color-brown`
+- `--mod-badge-subtle-background-color-celery`
+- `--mod-badge-subtle-background-color-chartreuse`
+- `--mod-badge-subtle-background-color-cinnamon`
+- `--mod-badge-subtle-background-color-cyan`
+- `--mod-badge-subtle-background-color-default`
+- `--mod-badge-subtle-background-color-fuchsia`
+- `--mod-badge-subtle-background-color-gray`
+- `--mod-badge-subtle-background-color-green`
+- `--mod-badge-subtle-background-color-indigo`
+- `--mod-badge-subtle-background-color-informative`
+- `--mod-badge-subtle-background-color-magenta`
+- `--mod-badge-subtle-background-color-negative`
+- `--mod-badge-subtle-background-color-notice`
+- `--mod-badge-subtle-background-color-orange`
+- `--mod-badge-subtle-background-color-pink`
+- `--mod-badge-subtle-background-color-positive`
+- `--mod-badge-subtle-background-color-purple`
+- `--mod-badge-subtle-background-color-red`
+- `--mod-badge-subtle-background-color-seafoam`
+- `--mod-badge-subtle-background-color-silver`
+- `--mod-badge-subtle-background-color-turquoise`
+- `--mod-badge-subtle-background-color-yellow`
+- `--mod-badge-subtle-label-icon-color`
+- `--mod-badge-workflow-icon-size`
+
+</details>
+
+### SWC
+
+<details>
+<summary>Attributes</summary>
+
+- `variant` - Badge variant (accent, neutral, informative, positive, negative, notice, fuchsia, indigo, magenta, purple, seafoam, yellow, gray, red, orange, chartreuse, celery, green, cyan, blue)
+- `fixed` - Fixed positioning (inline-start, inline-end, block-start, block-end)
+- `size` - Badge size (s, m, l, xl)
+
+</details>
+
+<details>
+<summary>Slots</summary>
+
+- Default slot - Text label of the badge
+- `icon` - Optional icon that appears to the left of the label
+
+</details>
+
+## Comparison
+
+### Visual comparison
+
+**Legacy Component:**
+
+<!-- Screenshot of legacy component will be added here -->
+
+**Spectrum 2 Component:**
+
+<!-- Screenshot of Spectrum 2 component will be added here -->
+
+### DOM structure changes
+
+<details>
+<summary>Spectrum Web Components:</summary>
+
+```html
+<sp-badge variant="informative" fixed="inline-start" size="m">
```

Comment:
I don't think the fixed or size props are set on the default badge WC

Reply:
Do we want it to definitely be markup for the default badge? It wasn't intentional for it to include variants and sizes and such, but I think it's kind of helpful to see how they could be applied.